src/data/polynomial/erase_lead.lean

```diff
@@ -3,8 +3,8 @@ Copyright (c) 2020 Damiano Testa. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Damiano Testa
 -/
-import data.polynomial.degree
 import data.polynomial.degree.trailing_degree
+import data.polynomial.inductions
```

Comment:
This is the main change in imports, that required all other imports to change as well, right?

Reply:
Yes, my idea was to get as quickly as possible to be able to prove the inductions and then import this file whenever it is needed.